rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -963,8 +963,15 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	m_emu_settings->EnhanceCheckBox(ui->debugConsoleMode, emu_settings::DebugConsoleMode);
 	SubscribeTooltip(ui->debugConsoleMode, tooltips.settings.debug_console_mode);
 
-	m_emu_settings->EnhanceCheckBox(ui->accuratePPUfma, emu_settings::AccuratePPUfma);
-	SubscribeTooltip(ui->accuratePPUfma, tooltips.settings.accurate_ppu_fma);
+	if (utils::has_fma3() || utils::has_fma4())
+	{
+	ui->accuratePPUfma->setEnabled(false);
```

Comment:
???

Reply:
What's the confusion?

Reply:
it's still not indented

Reply:
Is it good now?

Reply:
It's worse...
Idk what you're doing... But it's not working

Reply:
*Should* be good now?

Reply:
Yes. Looks good now